### PR TITLE
Add 2 fields to OutputFile class and full test coverage.

### PR DESCRIPTION
**Pull request by hacktobeer**
* Add extra OutputFile class fields output_path and source_file_id
* Add 100% test coverage + github workflow for pytest
* Remove _generate_file_name as it will always be a uuid
* Cleanup docstrings and formatting

### CODE REVIEW

**Comment by hacktobeer** [comment]
@berggren PTAL. Can you use this to plumb through the 2 extra fields in the mediator?

**Comment by hacktobeer** [comment]
Example of github action for Python Tests can be seen at https://github.com/hacktobeer/openrelik-worker-common/actions/runs/11081575550

File: openrelik_worker_common/utils.py [hunk 1]
```diff
@@ -169,8 +179,11 @@ def create_output_file(
         filename: The name of the output file (optional).
         file_extension: The extension of the output file (optional).
         data_type: The data type of the output file (optional).
+        orignal_path: The orignal path of the file (optional).
```
Comment (berggren) [inline openrelik_worker_common/utils.py:182]:
```suggestion
        original_path: The orignal path of the file (optional).
```

File: .github/workflows/python-test.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.12"]
```
Comment (berggren) [inline .github/workflows/python-test.yaml:11]:
```suggestion
        python-version: ["3.10", "3.12"]
```

Comment (berggren) [reply .github/workflows/python-test.yaml:11]: Adding 3.10 as this is common in some environments and is the default version for workers.

**Review by berggren — APPROVED** [review]
LGTM, I just added 3.10 to the test runs as well so we cover that version.

**Pull request merged**